Mathlib/Order/Partition/Finpartition.lean

```diff
@@ -520,6 +520,8 @@ theorem card_parts_le_card (P : Finpartition s) : P.parts.card ≤ s.card := by
 
 section Atomise
 
+set_option synthInstance.maxHeartbeats 0 in -- Porting note: this is too slow
+open Classical in -- Porting note: haha
```

Comment:
Does using the `classical` tactic in the proof work instead?

Reply:
Nope, it's needed in the statement.